os_net_config/impl_nmstate.py

```diff
@@ -1833,6 +1840,34 @@ def parse_ovs_extra(self, ovs_extras, name, data):
             for cmd_map in cfg_val_pair:
                 self._ovs_extra_cfg_val(ovs_extra_cmd, cmd_map, data)
 
+    def parse_ovs_extra_for_iface(self, ovs_extras, iface_name, data):
+        """Parse ovs extra for interface
+
+        :param ovs_extras: given ovs extra as string
```

Comment:
Suggestion: I know we have the same docstring other places, but it should probably be clear that ovs_extras is a list and not a string.